tests/tacacs/utils.py

```diff
@@ -66,24 +66,31 @@ def log_exist(ptfhost, sed_command):
     pytest_assert(exist, "Not found data: {} in tacplus server log".format(data))
 
 
-def get_auditd_config_reload_line_count(duthost):
```

Comment:
>get_auditd_config_reload_line_count [](http://example.com/codeflow?start=4&length=35)

Let's keep original function, and create a new one, even the old one is not used right now.